xcvm/lib/core/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+authors = ["Composable Developers"]
+edition = "2021"
+homepage = "https://composable.finance"
+name = "xcvm-core"
+version = "0.1.0"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false }
+scale-info = { version = "2.1.1", default-features = false, features = [
+  "derive",
+] }
+schemars = { version = "0.8.1", optional = true }
+serde = { version = "1", default-features = false, features = [ "derive", "alloc" ] }
+fixed = { version = "1.15", default-features = false }
+num = { version = "0.4", default-features = false }
+
+[features]
+default = []
+std = ["dep:schemars"]
```

Comment:
I haven't seen this syntax yet though lol

Reply:
https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies